File: osb-mobile-main/src/features/pix/redux/models/request/changeAccountOperationLimitRequest.ts
import { ApiRequest } from '_config'

export interface ChangeAccountOperationLimitRequest extends ApiRequest {
  TaxId: string
  CompanyId: number
  OperationType?: number
  AccountOperationLimitType?: number
  AccountOperationLimitSubType?: number
  MinLimitValue: number
  MaxLimitValue: number
  Bank?: string
  BankBranch?: string
  BankAccount?: string
  BankAccountDigit?: string
}

File: osb-core-main/Platform/Business/Osb.Core.Platform.Business.Service/Models/Request/ChangeAccountOperationLimitRequest.cs
namespace Osb.Core.Platform.Business.Service.Models.Request
{
    public class ChangeAccountOperationLimitRequest : BaseRequest
    {
        public string TaxId { get; set; }
        public long CompanyId { get; set; }
        public int OperationType { get; set; }
        public int AccountOperationLimitType { get; set; }
        public int AccountOperationLimitSubType { get; set; }
        public decimal MinLimitValue { get; set; }
        public int MaxLimitValue { get; set; }
        public string Bank { get; set; }
        public string BankBranch { get; set; }
        public string BankAccount { get; set; }
        public string BankAccountDigit { get; set; }


    }
}